src/sentry/dynamic_sampling/rules/helpers/sliding_window.py

```diff
@@ -13,6 +13,28 @@
 # Sentinel value used to mark that an error happened when computing the sliding window sample rate for a specific
 # project.
 SLIDING_WINDOW_CALCULATION_ERROR = "sliding_window_error"
+# We want to keep the entry for 1 hour, so that in case an org is not considered for 1 hour, the system will fall back
+# to the blended sample rate.
+EXECUTED_CACHE_KEY_TTL = 60 * 60 * 1000
```

Comment:
Ideally this would be expressed as a factor on top of the task's schedule interval, but with how we configure cron schedules that's not easily possible. If you like, leave a comment here and on the task schedule that those two should be in some relation.